.github/workflows/robot-ui-e2e-tests.yml

```
@@ -0,0 +1,256 @@
+                    sudo npx playwright install-deps
+
+            -   name: Install docker-compose
+                run: |
+                    sudo curl -L "https://github.com/docker/compose/releases/download/2.12.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+                    sudo chmod +x /usr/local/bin/docker-compose
+            -   name: Install Robot tests
+                run: |
+                    git clone https://github.com/spryker/robotframework-suite-tests.git --single-branch --branch master robotframework-tests
+            -   name: Install Project
+                continue-on-error: false
+                run: |
+                    git clone https://github.com/spryker/docker-sdk.git ./docker
+                    docker/sdk boot -v deploy.ci.acceptance.mariadb.robot.yml
+                    sudo bash -c "echo '127.0.0.1 backend-api.at.spryker.local backend-api.de.spryker.local glue-backend.de.spryker.local glue-backend.at.spryker.local glue-storefront.de.spryker.local glue-storefront.at.spryker.local backend-gateway.at.spryker.local backend-gateway.de.spryker.local backoffice.at.spryker.local backoffice.de.spryker.local date-time-configurator-example.spryker.local glue.at.spryker.local glue.de.spryker.local yves.at.spryker.local yves.de.spryker.local mp.de.spryker.local mail.spryker.local mp.at.spryker.local' >> /etc/hosts"
+                    docker/sdk up -t -v
+            -   name: Custom commands
+                continue-on-error: false
+                run: |
+                    docker/sdk cli composer dump-autoload -o -a --apcu
+                    APPLICATION_STORE=DE docker/sdk console queue:worker:start --stop-when-empty
+                    APPLICATION_STORE=AT docker/sdk console queue:worker:start --stop-when-empty
+            -   name: Run Tests
+                id: run_tests
+                continue-on-error: true
+                run: |
+                    cd robotframework-tests
+                    rfbrowser init
+                    robot -v env:ui_b2c -v headless:true -v ignore_console:false -d results --exclude skip-due-to-issueORskip-due-to-refactoring --include group_tree -s '*'.tests.ui.b2c .
+                    touch results/time.txt && echo $(date) > results/time.txt
+            -   name: Rerun Failed Tests
+                if: steps.run_tests.outcome != 'success'
+                run: |
+                    cd robotframework-tests
+                    robot -v env:ui_b2c -v headless:true -v ignore_console:false -d results/rerun --runemptysuite --rerunfailed results/output.xml --output rerun.xml -s '*'.tests.ui.b2c .
+            -   name: Merge Test Results
+                if: always() && steps.run_tests.outcome != 'success'
+                run: |
+                    cd robotframework-tests
+                    rebot -d results --merge results/output.xml results/rerun/rerun.xml
+            -   name: Upload artifacts
+                if: always()
```

```suggestion
                if: failure()
```
@Nataliia-Popkova what do u think about skip upload of robot logs when the build passed successfully?

---

Good idea